sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/RsaKeyCryptographyClient.java

```diff
@@ -82,16 +85,24 @@ private KeyPair getKeyPair(JsonWebKey key) {
         try {
             transform = algo.createEncryptor(keyPair);
             return Mono.just(new EncryptResult(transform.doFinal(plaintext), (byte[]) null, algorithm));
-        } catch (InvalidKeyException | NoSuchAlgorithmException | NoSuchPaddingException | IllegalBlockSizeException | BadPaddingException e) {
+        } catch (InvalidKeyException
+            | NoSuchAlgorithmException
+            | NoSuchPaddingException
+            | IllegalBlockSizeException
+            | BadPaddingException e) {
             return Mono.error(e);
         }
     }
 
     @Override
-    Mono<DecryptResult> decryptAsync(EncryptionAlgorithm algorithm, byte[] cipherText, byte[] iv, byte[] authenticationData, byte[] authenticationTag, Context context, JsonWebKey jsonWebKey) {
+    Mono<DecryptResult> decryptAsync(EncryptionAlgorithm algorithm, byte[] cipherText, byte[] iv,
+                                     byte[] authenticationData, byte[] authenticationTag, Context context,
+                                     JsonWebKey jsonWebKey) {
 
         if (iv != null || authenticationData != null || authenticationTag != null) {
-            return Mono.error(new IllegalArgumentException("iv, authenticationData and authenticationTag parameters are not supported for Rsa decrypt operation"));
+            return Mono.error(new IllegalArgumentException(
+                "iv, "
```

Comment:
This is an interesting place to split the string. I'd try to fit as much as possible on the first line.